.github/CODEOWNERS

```diff
@@ -126,7 +126,7 @@ processor/schemaprocessor/                           @open-telemetry/collector-c
 processor/spanmetricsprocessor/                      @open-telemetry/collector-contrib-approvers @albertteoh
 processor/spanprocessor/                             @open-telemetry/collector-contrib-approvers @boostchicken @pmm-sumo
 processor/tailsamplingprocessor/                     @open-telemetry/collector-contrib-approvers @jpkrohling
-processor/transformprocessor/                        @open-telemetry/collector-contrib-approvers @Aneurysm9 @bogdandrutu @TylerHelmuth @kentquirk
+processor/transformprocessor/                        @open-telemetry/collector-contrib-approvers @bogdandrutu @TylerHelmuth @kentquirk
```

Comment:
Please keep same order with you first and me last :) since you are the main owner :)